features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -864,6 +864,7 @@ CREATE TABLE IDP_AUTHENTICATOR (
             NAME VARCHAR(255) NOT NULL,
             IS_ENABLED CHAR (1) DEFAULT '1',
             DISPLAY_NAME VARCHAR(255),
+            DEFINED_BY VARCHAR(255) NOT NULL,
```

Comment:
Defined by is always system and custom so there's no need to set a larger length like 255

Reply:
Addressed with commit 98816d3b7bdeebad908465279902613029f8bd59